test/runtests.jl

```diff
@@ -77,6 +77,26 @@ Agent8(id, pos; f1, f2) = Agent8(id, pos, f1, f2)
     f3::Float64 = 3.0
 end
 
+@agent Agent10 NoSpaceAgent begin
```

Comment:
please do not define agent types outside the test they are needed. I know this is what we do in the tests at the moment very often but it is very bad practice that we are slowly fixing.

Reply:
see https://github.com/JuliaDynamics/Agents.jl/issues/634